bool is_line_detected
int32 lateral_error
float32 angle_error
bool is_square_detected
int32 center_x1_error
int32 center_y1_error
bool is_circle_detected
int32 center_x2_error
int32 center_y2_error